example/test-make-vcow-swappable.json

```diff
@@ -2,5 +2,10 @@
   "cowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
   "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
   "atomsToTransfer": "123456000000000000000000",
-  "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
+  "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad",
+  "multiTokenMediator": "0x88ad09518695c6c3712AC10a214bE5109a655671",
+  "bridged": {
+    "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
```

Comment:
Why is the same vCow token as on mainnet?
isn't `0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB` the right address?

Reply:
You are right, fixed! Thanks for checking.